packages/x-date-pickers-pro/src/DesktopDateTimeRangePicker/DesktopDateTimeRangePicker.tsx

```diff
@@ -91,8 +91,9 @@ const rendererInterceptor = function rendererInterceptor<
         availableRangePositions: [rangePosition],
         view: !isTimeViewActive ? popperView : 'day',
         views: rendererProps.views.filter(isDatePickerView),
+        sx: [{ gridColumn: 1 }, ...finalProps.sx],
```

Comment:
We often do the following:


```
      sx={[...(Array.isArray(formControlSx) ? formControlSx : [formControlSx])]}
```

Are we sure `finalProps.sx` is an array here?